scripts/create_baseline_stubs.py

```diff
@@ -46,6 +46,8 @@ def get_installed_package_info(project: str) -> tuple[str, str] | None:
 
     Return (normalized project name, installed version) if successful.
     """
+    # Not using "uv pip freeze" because if this is run from a global Python,
+    # it'll mistakenly list the .venv's packages.
```

Comment:
```suggestion
    # it'll mistakenly list the .venv's packages. Also, not all contributors have `uv`.
```

Reply:
We *do* assume uv in all scripts though. Because of: https://github.com/python/typeshed/blob/28106bcb73835713231095f108f0994845092403/requirements-tests.txt#L22

What we don't assume is that `uv run` or a local venv called `.venv` will be used to run the scripts.